.github/workflows/continuous-integration.yml

```diff
@@ -118,7 +118,6 @@ jobs:
           set -o pipefail && \
           NSUnbufferedIO=YES xcodebuild \
             -scheme aws-sdk-swift-protocol-tests-Package \
-            -testPlan ProtocolTestPlan \
```

Comment:
This protocol test plan is deleted (see below).  Its only purpose was to get credentials into Xcode test runs, now this is done in protocol test setup.